packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/AnonymousBodyRequest.Serialization.cs

```diff
@@ -192,8 +192,7 @@ internal static AnonymousBodyRequest DeserializeAnonymousBodyRequest(JsonElement
             IList<int> requiredNullableList = default;
             ModelWithBaseModelWithoutRequired baseWithoutRequired = default;
             ModelWithBaseModelWithRequired baseWithRequired = default;
-            IDictionary<string, BinaryData> serializedAdditionalRawData = default;
-            Dictionary<string, BinaryData> rawDataDictionary = new Dictionary<string, BinaryData>();
+            IDictionary<string, BinaryData> serializedAdditionalRawData = new ChangeTrackingDictionary<string, BinaryData>();
```

Comment:
I changed how this is initialized, but this actually changed nothing in the runtime.
previously we always instantiate a new dictionary, and now we are always doing that.
As a future work item, this might need to be lazy loaded - when there is no extra properties, we have a null raw data.